src/Sql/Sql.Sdk/Generated/SdkInfo_SqlManagementClient.cs

```diff
@@ -96,7 +96,7 @@ internal static partial class SdkInfo
                 new Tuple<string, string, string>("Sql", "ServerAzureADOnlyAuthentications", "2020-11-01-preview"),
                 new Tuple<string, string, string>("Sql", "ServerBlobAuditingPolicies", "2021-11-01-preview"),
                 new Tuple<string, string, string>("Sql", "ServerConfigurationOptions", "2022-08-01-preview"),
-                new Tuple<string, string, string>("Sql", "ServerDevOpsAuditSettings", "2020-11-01-preview"),
+                new Tuple<string, string, string>("Sql", "ServerDevOpsAuditSettings", "2022-05-01-preview"),
```

Comment:
We don't support updating the api-version directly from the generated sdk files. Please update the api-version from README.md and then re-generate SDK code.

Reply:
I did not update this manually;This file is generated. I updated the readme file. [here](https://github.com/Azure/azure-powershell/pull/21898/files#diff-c93755d6b1c06176d1b3246b84a795992e0495756584650faf37522e2f3149ba).